ptx/sec_deriv_intro.ptx

```diff
@@ -186,7 +186,17 @@
         <caption>The function <m>f(t)</m> and its secant line corresponding to <m>t=2</m> and <m>  t=3</m></caption>
           <image xml:id="img_derivfalling">
           <!-- START figures/fig_derivfalling.tex -->
-            <description></description>
+          <shortdescription>
+            A decreasing concaved down graph of function  <m>f(t)</m> with its secant line corresponding to <m>t=2 </m> and <m>t=3</m>.
+            </shortdescription>
+              <description>
+              <p>
+              The graph starts at the origin and is drawn on the first quadrant. 
+              The horizontal t-axis increases by 1 going right. The vertical y-axis increases by 50 moving upwards. 
+              The graph has a decreasing concave down shape, gently declinig from points <m>(0, 150)</m> to <m>(3,0)</m>.
```

Comment:
Typo: 'declinig'.

I'm not sure if "increases by 1", and "increases by 50" are clear. You could say that the scale is 50 to one, or perhaps that tick marks on the y axis, representing an increase of 50 units, are spaced the same as tick marks on the x axis, which represent a change of 1 unit..

Reply:
You didn't fix this typo yet.